com.unity.mobile.notifications/Tests/Runtime/iOS/iOSNotificationTests.cs

```diff
@@ -34,10 +75,14 @@ public void AfterEachTest()
         receivedNotificationCount = 0;
         lastReceivedNotification = null;
     }
+#endif
 
     [UnityTest]
     public IEnumerator SendSimpleNotification_NotificationIsReceived()
     {
+#if UNITY_EDITOR
```

Comment:
There's a better a way, please use UnityPlatform attribute and specify the platforms where this test should run

Reply:
You can actually specify UnityPlatform  on top of class, if this whole class is for iOS tests only

Reply:
Good one, didn't knew about that. Fixed.